### PR TITLE
Clarify Reason record literal syntax

### PR DESCRIPTION
**Pull request by yawaramin**
Also warn about code example tables in readme.

### CODE REVIEW

**Comment by chenglou** [comment]
Hummmm I'm not sure this is the best intro example for record. It's a bit up-front about a case that newcomers won't encounter just yet. Maybe move it after the description and do "If the record you're trying to access is somewhere else, then ..."?

**Comment by yawaramin** [comment]
@chenglou I see what you mean. Let me try out your suggestion.

**Comment by hcarty** [comment]
The third example (`{ Module.field1; field2; ...}`) is probably worth highlighting first as it is the safest example.  The other two perform a local open of the parent module which affects the environment beyond exposing the record field names.

File: docs/javaScriptCompared.html [hunk 1]
```diff
@@ -322,9 +322,91 @@
...
+  /&ast;
+  Alternatively, using field name punning,
+  let make id name => { id, name };
+  &ast;/
+};</pre>
+    </td>
+  </tr>
+</table>
+
+When the record type is declared in a different modue, you have to give Reason a
```
Comment (Schmavery) [inline docs/javaScriptCompared.html]: modue

**Comment by yawaramin** [comment]
Bump :-)

**Comment by chenglou** [comment]
Looks good now. Thanks!

**Pull request merged**